x-pack/plugins/cases/public/components/add_comment/index.test.tsx

```diff
@@ -72,6 +72,13 @@ describe('AddComment ', () => {
   });
 
   it('should render spinner and disable submit when loading', async () => {
+    createAttachmentsMock.mockImplementation(
+      () =>
+        new Promise((resolve) => {
+          /* empty promise that never resolves to test loading state */
```

Comment:
The test is checking the loading state, but it seems that with the library update the loading state is gone before the test could check if it is there. I fixed it by make the loading never complete, so spinner doesn't go away.

Reply:
Rather than having an unresolved promise I think we should rather mock

```
  useCreateAttachmentsMock.mockReturnValue({
    isLoading: false,
    mutateAsync: createAttachmentsMock,
  });
  ```

Reply:
I think it just tries to mock one of the functions returned by `useCreateAttachments`, in this case, `createAttachments` which is the only one used in the `AddFile` component.

> ut didn't figure out how to do it neatly since I'd want to mock only for one test and keep the original for all others, right?

Hmm, that's a good point, it would require bigger changes. If it's ok that I push to this branch I can try fixing it later 🙌

Reply:
yes, please push 🙏  super welcome

Reply:
Fixed in 989fb10d08446f5b75d295cab337d438bd13e7e3